package.json

```diff
@@ -50,8 +50,21 @@
 			"activitybar": [
 				{
 					"id": "claude-dev-ActivityBar",
-					"title": "Cline",
-					"icon": "assets/icons/icon.svg"
+					"title": "Cline (⌘+')",
```

Comment:
@pashpashpash This goes against what we had discussed before but sadly this naming comes from package.json and title and tooltip match so we can't have a separate tooltip.


<img width="739" alt="image" src="https://github.com/user-attachments/assets/ded958cc-93c5-48a3-a985-1a44fc5bcfba" />